tech_reports/LLMs/llms.md

```diff
@@ -37,6 +37,17 @@ Authors:
         - [4.10.4.2 Large Matmuls](#41042-large-matmuls)
 
 ## 1. Overview
+What’s the best way to bring up high-performance multi-chip models on Tenstorrent hardware with the Metal stack? We continue to try to answer this question; here are our current best practices, tips and workarounds.
+
+What you need:
+
+* Access to TT hardware. This guide is specifically for bringing models up on wormhole, so whilst most of this advice applies equally to grayskull it is very WH (wormhole)-centric.
+* Good grasp of PyTorch and transformers. We will not be covering any basics here. For example, this document assumes you understand what a kv-cache is and get the difference between prefill (reading tokens and generating the kv-cache entries) and decode (auto-regressively generating new tokens one at a time). Beginner tutorials will follow, for now this is to help experts get up to speed deploying LLMs on Metal.
```

Comment:
`We will not be covering any basics here.` -> `This document will only skim some basics.`

Reply:
Yes. I agree with comment. We are briefly describing kv-cache, decode/prefill in other sections, but we do get into details.

Reply:
I found this is a great intro. Maybe we should link it.
https://levelup.gitconnected.com/building-llama-3-from-scratch-with-python-e0cf4dbbc306